packages/expo-cli/src/appleApi/provisioningProfile.ts

```diff
@@ -12,15 +22,51 @@ export class ProvisioningProfileManager {
     this.ctx = appleCtx;
   }
 
-  async create(bundleIdentifier: string, metadata: any = {}): Promise<ProvisioningProfile> {
+  async useExisting<T extends DistCert>(
+    bundleIdentifier: string,
+    provisioningProfile: ProvisioningProfile,
+    distCert: T
+  ): Promise<ProvisioningProfile> {
+    const args = [
+      'use-existing',
+      this.ctx.appleId,
+      this.ctx.appleIdPassword,
+      this.ctx.team.id,
+      String(this.ctx.team.inHouse),
+      bundleIdentifier,
+      provisioningProfile.provisioningProfileId,
+      distCert.distCertSerialNumber || '__last__',
```

Comment:
is fallback to `__last__` correct behavior here?
I'm not 100% sure but older dist cert and provided manually might not have a serial number, we should probably generate it
https://github.com/expo/expo-cli/blob/master/packages/xdl/src/detach/PKCS12Utils.ts#L4

I'm not exactly sure where to handle that, it could be done every place like this where we relly on serial number or on get api calls.